### ISSUE TITLE
Dual APU Support

### ISSUE THREAD

**Issue by TheRealHamtaro126** [BleuBleu/FamiStudio#150]
For both DK3 (Donkey Kong 3) Arcade version as well as and some Hummer Team systems (OneBus/VT).

You can read about the dual-APU setup in OneBus VT systems by Hummer Team here:
https://www.nesdev.org/wiki/VT02%2B_Sound

The DK3 engine is similar, but the second APU may be relocated, I dunno about that, There is no disassembly on DK3's audio engine currently as far as I know.

**Comment by BleuBleu** [comment]
Yeah that's something id really like to do down the line. Probably not super short term since i am busy finishing 4.0.0 right now. Maybe later. Ill keep the issue open.

-Mat

**Comment by TheRealHamtaro126** [comment]
Thank you for your consideration, Make sure the "Enhancement" tag is on so people know it is considered on Github.
